ui/component/common/file-list.jsx

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from 'react';
+import * as ICONS from 'constants/icons';
+import Icon from 'component/common/icon';
+import classnames from 'classnames';
+import { useRadioState, Radio, RadioGroup } from 'reakit/Radio';
+
+type Props = {
+  files: Array<WebFile>,
+  onChange: (WebFile | void) => void,
+};
+
+function FileList(props: Props) {
+  const { files, onChange } = props;
+  const radio = useRadioState();
+
+  const getFile = (value?: string) => {
+    if (files && files.length) {
+      return files.find((file: WebFile) => file.name === value);
+    }
+  };
+
+  React.useEffect(() => {
+    if (radio.stops.length) {
+      if (!radio.currentId) {
+        radio.first();
+      } else {
+        const first = radio.stops[0].ref.current;
+        // First auto-selection
+        if (first && first.id === radio.currentId && !radio.state) {
+          const file = getFile(first.value);
+          // Update state and select new file
+          onChange(file);
+          radio.setState(first.value);
+        }
+
+        if (radio.state) {
+          // Find selected element
+          const stop = radio.stops.find(item => item.id === radio.currentId);
+          const element = stop && stop.ref.current;
+          // Only update state if new item is selected
+          if (element && element.value !== radio.state) {
+            const file = getFile(element.value);
+            // Sselect new file and update state
+            onChange(file);
+            radio.setState(element.value);
+          }
+        }
+      }
+    }
+  }, [radio, onChange]);
+
+  return (
+    <div className={'file-list'}>
+      <RadioGroup {...radio} aria-label="files">
```

Comment:
I would like avoid creating another radio button style. Can you use the current radio buttons? Or update `<FormField type="radio" />` (And maybe creating `<FormFieldRadioGroup />`)

Reply:
Where are the radio styles? I can't find it

Reply:
I can't use the current `FormField` component with `reakit`, but now it has the same styles